app/src/main/java/ru/vssemikoz/newsfeed/MainActivity.java

```diff
@@ -51,14 +61,33 @@ void initRecView(){
     }
 
     void initRecViewData(){
```

Comment:
Let's make your methods private since now they're package-protected and may be accessed outside of this activity. This will make your scope dirty and there may be potential security issues once.